index.js

```diff
@@ -83,7 +131,16 @@ function countByRating() {}
       // Toy Story 4
     };
  */
-function findById() {}
+function findById(movies, id) {
+  let foundMovie = null;
+
+  for (let i = 0; i < movies.length; i ++){
+    if (movies[i].imdbID === id){
+      foundMovie = movies[i];
+    }
```

Comment:
Currently this function will continue to iterate even if we've found the movie.  We should add a `break` or return `foundMovie` once we find it.  Imagine we have 10,000,000 movies and we find our match on the first iteration.  We would still loop through and check _ever other index_